oauth2client/contrib/django_util/decorators.py

```diff
@@ -52,6 +53,11 @@ def requires_default_scopes(request):
     def curry_wrapper(wrapped_function):
         @wraps(wrapped_function)
         def required_wrapper(request, *args, **kwargs):
+            if django_util.oauth2_settings.storage_model is not None:
+                if not request.user.is_authenticated():
+                    return shortcuts.redirect(
+                        '{}?next={}'.format(
+                            settings.LOGIN_URL, request.path))
```

Comment:
request.path will likely need to be urlencoded, does django's `shortcuts.redirect` not provide a way for you to do that?